app/uk/gov/hmrc/homeofficesettledstatusstubs/models/StatusResponse.scala

```diff
@@ -18,6 +18,7 @@ package uk.gov.hmrc.homeofficesettledstatusstubs.models
 
 import play.api.libs.json.{Format, Json}
 
+//todo split this in to 2 objects for Error and success
```

Comment:
Do we maybe want to do this under this task?

Reply:
:+1: